docs/en/source/post_queries.rst

```diff
@@ -0,0 +1,115 @@
+************
+POST QUERIES
+************
+
+The reason to provide a way to query NGSI_LD entities via POST, is using GET:
```

Comment:
.... is that using GET has the following issue:

Reply:
Done